pints/_mcmc/__init__.py

```diff
@@ -423,13 +423,32 @@ def run(self):
 
             # Calculate scores
             fxs = evaluator.evaluate(xs)
-
+            
+            a_none = False
             # Perform iteration(s)
             if self._single_chain:
                 samples = np.array([
                     s.tell(fxs[i]) for i, s in enumerate(self._samplers)])
+                for i, sample in enumerate(samples):
```

Comment:
This breaks parallelism

Reply:
@MichaelClerx Ok. Or I can just grab that code and use it here?

Reply:
@MichaelClerx I looked at your code and not sure it'll work for this example. Unlike HMC where, inbetween full updates, None is always returned, in DRAM, whether or not None is returned is stochastic. So, I think we have to break parallelism there because, whereas one chain may be None, another may not be. What do you think?